source/default/_patterns/02-molecules/vue-widget/demo/vue-html-example.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <form 
-    id="vue-html-example-2" 
+    id="vue-html-example-2"
+    onsubmit="return false"
```

Comment:
@jengrace tell me about `return false` and accessibility, I'm learning something new here.